nero_view.py

```diff
@@ -11,8 +12,14 @@
 import platform
 import time
 from models.model import Model
+from header import Header
+import os
 
 customtkinter.set_appearance_mode("dark")
+
+if platform.platform()[0:5] == "Linux":
```

Comment:
why put this between the two customtkinter functions that make a lot of sense together

Reply:
because on the pi, the autoboot doesnt start without this line